pysat/_files.py

```diff
@@ -180,6 +182,32 @@ def __init__(self, sat, manual_org=False, directory_format=None,
                 # couldn't find stored info, load file list and then store
                 self.refresh()
 
+    def __repr__(self):
+        out_str = "".join(["Files(", self._sat.__repr__(), ", manual_org=",
+                           "{:}, directory_format=".format(self.manual_org),
+                           self.directory_format, ", update_files=",
```

Comment:
self.directory format needs to be printed as a string, aka the string itself needs internal quotes. the %r style

Reply:
Fixed in latest commit.